### ISSUE TITLE
[knx] DPT 9.004 not working in KNX 2.4.0 Binding (but was in 1.X)

### ISSUE THREAD

**Issue by Ronny-GitHub** [openhab/openhab2-addons#6109]
For the KNX 1.X binding there was in issue [KNX-Binding: Numerical values with DPT 9.004 are not shown](https://github.com/openhab/openhab1-addons/issues/123) reported, which is now closed. 

I noticed that brightness changes my motion detector is reporting are not reported on my sidemap anymore.
 
Here is an example channel definition:
`Type number        : Motion_OG_Kueche_Brightness  "Motion Brightness [%.1f Lux]"      [ ga="<9.004:15/1/7" ]`

As there is a stored value in my persistence db I did not notice until now when I was wondering why the values always stay the same and do not change, even when I point a flashlight on the sensor.

I can see in ETS that different values are reported, but they never make it to the item.
As this applies to all items I have set up, I think this must be an issue with the KNX binding.

Here are the (german) original item setting:
`Number Bewegung_OG_Kueche_Helligkeit            "Bewegung Küche Helligkeit [%.1f Lux]"      <presence> (Bewegung_OG)                  { channel="knx:device:bridge:Bewegung_Device:Bewegung_OG_Kueche_Helligkeit" }`
and the (german) thing setting:
`Type number        : Bewegung_OG_Kueche_Helligkeit            "Bewegung Küche Helligkeit [%.1f Lux]"      [ ga="<9.004:15/1/7" ]`

If you feel this is not a coding, but a setup bug (by me) please let me know and feel free to close the ticket.

**Comment by kschn** [comment]
I think it's a coding bug. Instead of writing
`[ga="<9.004:15/1/7"]`
you should write
`[ga="9.004:<15/1/7"]`

**Comment by Ronny-GitHub** [comment]
OK - this is a difference between the KNX 1.X binding and the KNX 2.4.0 binding. As I now saw all the examples are written like this I tried it out and it seems to work.
Strange thing is that I have some other channels which had like DPT 9.001 / 5.001 or 12.001 which I believe are working OK.

Since this seems to be the new convention, I will change my settings and the two issues I have opened can be closed.
Thanks for the quick feedback.
